schema.graphql

```diff
@@ -882,6 +894,7 @@ type UserNode implements Node {
   societies(offset: Int, before: String, after: String, first: Int, last: Int, id: ID, name: String, slug: String, userHasPermission: String): SocietyNodeConnection!
   bookings(offset: Int, before: String, after: String, first: Int, last: Int, createdAt: DateTime, updatedAt: DateTime, status: String, reference: String, user: ID, creator: ID, performance: ID, adminDiscountPercentage: Float, expiresAt: DateTime, id: ID, statusIn: [String], search: String, checkedIn: Boolean, active: Boolean, expired: Boolean, orderBy: String): BookingNodeConnection!
   createdBookings(offset: Int, before: String, after: String, first: Int, last: Int, createdAt: DateTime, updatedAt: DateTime, status: String, reference: String, user: ID, creator: ID, performance: ID, adminDiscountPercentage: Float, expiresAt: DateTime, id: ID, statusIn: [String], search: String, checkedIn: Boolean, active: Boolean, expired: Boolean, orderBy: String): BookingNodeConnection!
+  ticketsChecked(offset: Int, before: String, after: String, first: Int, last: Int): TicketNodeConnection!
```

Comment:
restrict this just to the ticket object and not deeper?

Reply:
I think this could be a bit clearer - is it checked tickets that belong to the user, or tickets that have been checked in by the user. But also I think maybe lets not expose this for now? I don't see us needing this any time soon.